src/common/Select/Select.js

```diff
@@ -50,7 +50,7 @@ const Select = ({
 
   return (
     <div
-      className={`select ${className} ${isOpen ? ' active' : ''}`}
+      className={`select ${className} ${isOpen ? 'active' : ''}`}
```

Comment:
Maybe you can use the `classnames` package here to conditionally add the `active` CSS class name.